### PR TITLE
View order page: Display correction - 2

### LINKED ISSUE #18236: [ORDER PAGE] Display correction - 2
New front-end enhancements to improve the order page
## Part 2 

- [x] In the mobile view, the product block is on the bottom of the page, below the message block. 

To prevent merchants from scrolling down the page to check their order, we should move up the product block below the customer block.

- [x] The currency selector in the payment block takes up the entire width of the column which reduces the size of the next column. The selector should have the size of its contents.

- [x] when adding a product, the buttons are not aligned
image
- [x] when adding a product, the quantity background color is not round
- [x] on tablet view, add a product and add a discount overlap
image
- [x] the contour of the status block is not quite like the others one: radius and shadow are missing.
- [x] To make tabs from the status, carrier... block scrollable instead of overlapping tabs on mobile view
- [x] The spacing between the vertical blocks must be 30px
- [x] Order listing, we still have the spacing from the hidden dropdown button in the status label

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | Some minor visual corrections on the order view page and order lists
| Type?         | improvement
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #18236 .
| How to test?  | Please rebuild assets. Go on the issue and point by point you need to verify than everything is correctly fixed.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/18329)

### CODE REVIEW

**Review by matks — DISMISSED** [review]

**Review by Progi1984 — DISMISSED** [review]

**Review by atomiix — DISMISSED** [review]

**Comment by Robin-Fischer-PS** [comment]
Hi @NeOMakinG !

- In the mobile view, the product block is on the bottom of the page, below the message block. To prevent merchants from scrolling down the page to check their order, we should move up the product block below the customer block : :x: _The product block is above customer block instead of below._
- The currency selector in the payment block takes up the entire width of the column which reduces the size of the next column. The selector should have the size of its contents. :heavy_check_mark: 
- when adding a product, the buttons are not aligned :question: Could not reproduce without the PR
- when adding a product, the quantity background color is not round :question: Could not reproduce without the PR
- on tablet view, add a button and add a discount overlap : :question:  Same as below, isn't it ?
- on tablet view, add a product and add a discount overlap :heavy_check_mark: 
- the contour of the status block is not quite like the others one: radius and shadow are missing. :question: I don't see any difference so I'm not sure it's OK.
- To make tabs from the status, carrier... block scrollable instead of overlapping tabs on mobile view :heavy_check_mark: 
- The spacing between the vertical blocks must be 30px :question: : Between which blocks ?
- Order listing, we still have the spacing from the hidden dropdown button in the status label :heavy_check_mark:

**Comment by NeOMakinG** [comment]
- when adding a product, the buttons are not aligned :question: Product with combination/custom
- when adding a product, the quantity background color is not round :question: Add a product with more than 1 quantity (tested with 10)
- on tablet view, add a button and add a discount overlap : Before this PR buttons were going under, now they stay aligned on every screens
- the contour of the status block is not quite like the others one: radius and shadow are missing. There are now a shadow and radius on the status block, like the others block, before, there was no radius and shadows
- The spacing between the vertical blocks must be 30px : Every blocks should be spaced from the others by 30px, that looks fine right now

Thanks @Robin-Fischer-PS, pushing the modification for the product block

**Review by Progi1984 — DISMISSED** [review]

**Review by jolelievre — DISMISSED** [review]

**Review by matks — DISMISSED** [review]

**Comment by khouloudbelguith** [comment]
Hi @NeOMakinG,

Can you please resolve conflicts.

Thanks!

**Review by PierreRambaud — DISMISSED** [review]

**Comment by NeOMakinG** [comment]
@khouloudbelguith, this is assets, not related to this PR, it doesn't block QA, will rebase after QA tests :)

**Comment by SD1982** [comment]
LGTM Thanks @NeOMakinG !

**Review by Progi1984 — APPROVED** [review]

File: src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/products.html.twig [hunk 1]
```diff
@@ -133,7 +133,7 @@
           </ul>
         </nav>
       </div>
-      <div class="col-md-6 text-right discount-action">
+      <div class="col-xl-8 text-right discount-action">
```
Comment (jolelievre) [inline src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/products.html.twig:136]: Shouldn't it be `col-md-8` ? This means they won't have the same behavior depending on the screen size?

Comment (NeOMakinG) [reply src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/products.html.twig:136]: This is to fix one of points of this PR

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
Thank you @NeOMakinG

**Pull request merged**